web/frontend/src/pages/form/components/FormForm.tsx

```diff
@@ -47,6 +49,24 @@ const FormForm: FC<FormFormProps> = () => {
 
   const { MainTitle, Scaffold } = conf;
   const regexPattern = /[^a-zA-Z0-9]/g;
+
+  const [postError, setPostError] = useState(null);
```

Comment:
You must use those to display an error in case something went wrong with the authorization update.